### PR TITLE
Add missing quote marks to the DALI_AFFINITY_MASK usage example

### PR DESCRIPTION
**Pull request by JanuszL**
- the example showing how to set the DALI_AFFINITY_MASK environment
  the variable is missing ""around the values

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation*)

## Description:
- the example showing how to set the DALI_AFFINITY_MASK environment
  the variable is missing ""around the values

### Affected modules and functionalities:
- Performance Tuning docs section

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

Relates to https://github.com/NVIDIA/DALI/issues/4018

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Review by szalpal — APPROVED** [review]

File: docs/advanced_topics_performance_tuning.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ An example:
 .. code-block:: bash
 
   num_threads=5
```
Comment (mzient) [inline docs/advanced_topics_performance_tuning.rst:33]: What is this doing here? It's not set in the shell, is it?

Comment (JanuszL) [reply docs/advanced_topics_performance_tuning.rst:33]: It is just to show the evn conditions.

**Comment by JanuszL** [comment]
!build

**Pull request merged**